cpp/bench/ann/src/raft/raft_benchmark.cu

```diff
@@ -272,5 +272,13 @@ REGISTER_ALGO_INSTANCE(std::uint8_t);
 
 #ifdef ANN_BENCH_BUILD_MAIN
 #include "../common/benchmark.hpp"
-int main(int argc, char** argv) { return raft::bench::ann::run_main(argc, argv); }
+int main(int argc, char** argv)
+{
+  rmm::mr::cuda_memory_resource cuda_mr;
+  // Construct a resource that uses a coalescing best-fit pool allocator
+  rmm::mr::pool_memory_resource<rmm::mr::cuda_memory_resource> pool_mr{&cuda_mr};
+  rmm::mr::set_current_device_resource(
```

Comment:
Since we're creating the pool memory resource here now - should we remove creating the pool in individual benchmarks like https://github.com/rapidsai/raft/blob/0d199f9d37cdf3921f04e4b657b5f3fe302cd106/cpp/bench/ann/src/raft/raft_cagra_wrapper.h#L59-L61 ?

Reply:
Ah yes, thanks for reminding me to remove these.